.github/workflows/release.yml

```diff
@@ -19,7 +19,6 @@ jobs:
           publish_to_binaries: true
           distribute: true
           attach_artifacts_to_github_release: true
-          run_rules_cov: true
```

Comment:
@tomverin Should we update all actions to @v4 as well? There was some random message on #team-lang about it.

Reply:
@mary-georgiou-sonarsource Let's update it everywhere from `@v3` to `@v4`, it worked on the VB6 one.